Dockerfile

```diff
@@ -1,26 +1,17 @@
-FROM metacell/jupyter-neuron:latest
+FROM frodriguez4600/jupyter-neuron:v7.8.0
 USER $NB_USER
 
-ARG netpyneuiBranch=development 
-ENV netpyneuiBranch=${netpyneuiBranch}  
-RUN echo "$netpyneuiBranch";
+ARG branch=development 
+RUN echo "$branch";
 
-ARG INCUBATOR_VER=unknown
-RUN /bin/bash -c "INCUBATOR_VER=${INCUBATOR_VER} source activate snakes && pip install netpyne_ui"
-RUN /bin/bash -c "source activate snakes && jupyter nbextension enable --py jupyter_geppetto"
-RUN /bin/bash -c "source activate snakes &&  jupyter serverextension enable --py jupyter_geppetto"
-RUN /bin/bash -c "source activate snakes && jupyter nbextension enable --py widgetsnbextension"
+ENV INSTALATION_FOLDER=/home/jovyan/work/NetPyNE-UI
+WORKDIR /home/jovyan/work
 
-WORKDIR /home/jovyan
-RUN git clone --branch v0.5 https://github.com/Neurosim-lab/netpyne_workspace
-WORKDIR /home/jovyan/netpyne_workspace
+COPY --chown=1000:1000 . NetPyNE-UI
+WORKDIR ${INSTALATION_FOLDER}/utilities
 
-# Uncomment to run travis using this Dockerfile
-# Clone the source code and creates a symlink to the test folder
-WORKDIR /home/jovyan/work
-RUN wget https://github.com/MetaCell/NetPyNE-UI/archive/$netpyneuiBranch.zip -q
-RUN unzip $netpyneuiBranch.zip 
-WORKDIR /home/jovyan/netpyne_workspace
-RUN ln -sfn /home/jovyan/work/NetPyNE-UI-$netpyneuiBranch/netpyne_ui/tests tests
+RUN python install.py branch $branch
+
+WORKDIR ${INSTALATION_FOLDER}
 
-CMD /bin/bash -c "source activate snakes && exec jupyter notebook --debug --NotebookApp.default_url=/geppetto --NotebookApp.token='' --library=netpyne_ui --NotebookApp.disable_check_xsrf=True"
+CMD /bin/bash -c "jupyter notebook --NotebookApp.default_url=/geppetto --NotebookApp.token='' --library=netpyne_ui --NotebookApp.disable_check_xsrf=True"
```

Comment:
question, why aren't we using conda inside the container as before? Just curious